src/styles/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  max-width: 200px;
+  padding: 30px 14px 14px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  &:hover {
```

better to leave empty line before nested selector
```suggestion
  justify-content: center;

  &:hover {
```